### PR TITLE
Add rosdep key for Python 'aioraven'

### PR DESCRIPTION
**Pull request by cottsay**
Test test test...

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -29,6 +29,10 @@ adafruit-pca9685-pip:
   ubuntu:
     pip:
       packages: [adafruit-pca9685]
+aioraven:
+  '*':
+    pip:
+      packages: [aioraven]
```
Comment (cottsay-alt) [inline rosdep/python.yaml:35]: Keys which contain only pip rules should have the '-pip' suffix.

**Review by cottsay-alt — CHANGES_REQUESTED** [review]
This is an automated review.

For changes related to rosdep:
* ✅ No EOL rules were added
* 📝 Not all rules could be verified programmatically
* ❌ Rules exclusively for pip packages should have the '-pip' suffix and should be placed in 'python.yaml'

File: rosdep/python.yaml (hunk 1 above)
Comment (cottsay-alt) [inline rosdep/python.yaml:35]: Keys which contain only pip rules should have the '-pip' suffix.

**Review by cottsay-alt — CHANGES_REQUESTED** [review]
This is an automated review.

For changes related to rosdep:
* ✅ No EOL rules were added
* 📝 Not all rules could be verified programmatically
* ❌ Rules exclusively for pip packages should have the '-pip' suffix and should be placed in 'python.yaml'

File: rosdep/python.yaml [hunk 2]
```diff
@@ -29,6 +29,10 @@ adafruit-pca9685-pip:
   ubuntu:
     pip:
       packages: [adafruit-pca9685]
+aioraven:
+  '*':
+    pip:
```
Comment (cottsay) [inline rosdep/python.yaml:34]: Another comment?

File: rosdep/python.yaml (hunk 1 above)
Comment (cottsay-alt) [inline rosdep/python.yaml:35]: Keys which contain only pip rules should have the '-pip' suffix.

**Review by cottsay-alt — CHANGES_REQUESTED** [review]
This is an automated review.

For changes related to rosdep:
* ✅ No EOL rules were added
* 📝 Not all rules could be verified programmatically
* ❌ Rules exclusively for pip packages should have the '-pip' suffix and should be placed in 'python.yaml'

File: rosdep/python.yaml (hunk 1 above)
Comment (cottsay) [reply rosdep/python.yaml:35]:
```suggestion
aioraven-pip:
  '*':
    pip:
      packages: [aioraven]
```

Comment (cottsay-alt) [inline rosdep/python.yaml:35]: Keys which contain only pip rules should have the '-pip' suffix.

```suggestion
aioraven-pip:
  '*':
    pip:
      packages: [aioraven]
```

**Review by cottsay-alt — CHANGES_REQUESTED** [review]
This is an automated review.

For changes related to rosdep:
* ✅ No EOL rules were added
* 📝 Not all rules could be verified programmatically
* ❌ Rules exclusively for pip packages should have the '-pip' suffix and should be placed in 'python.yaml'

**Pull request closed without merging**